### PR TITLE
1711_MattMiltonBE

### PR DESCRIPTION
**Pull request by mgmilton**
Tests aren't the best, but it's fully functional.

### CODE REVIEW

File: app/models/snack.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Snack < ApplicationRecord
+  validates_presence_of :name, :price
+  has_many :machine_snacks, dependent: :nullify
+  has_many :machines, through: :machine_snacks
+
+  def to_dollar
+    (price/100).round(5)
```
Comment (icorson3) [inline app/models/snack.rb]: there is a handy view helper called `number_to_currency` that makes currency a lot more straight forward.

**Comment by icorson3** [comment]
Nice job overall @mgmilton...would think about more robust testing including making a join object to really test the functionality. Allowing some edge cases and model testing could also help...in the future, I would spend more time beefing up the tests and less time working on functionality that was not in the spec. 🎉
